src/config.js

```diff
@@ -25,6 +25,7 @@ import path from 'path'
 export function resetConfigDefaults () {
 	return {
 		command: process.platform === 'win32' ? (process.env.COMSPEC || 'cmd.exe') : (process.env.SHELL || '/bin/sh'),
+		autoCommand: true,
```

Comment:
This should default to`false` otherwise the command setting will be automatically changed for anyone who updates x-tetminal to the next version.

Reply:
Good point!

Reply:
Fixed in https://github.com/bus-stop/x-terminal/pull/128/commits/67443755f3cf31d2a48b3023288c5380d3de6665